docs/reference/quickstart/full-text-filtering-tutorial.asciidoc

```diff
@@ -0,0 +1,588 @@
+{"index":{"_id":"2"}}
+{"title":"Spicy Thai Green Curry: A Vegetarian Adventure","description":"Dive into the flavors of Thailand with this vibrant green curry. Packed with vegetables and aromatic herbs, this dish is both healthy and satisfying. Don't worry about the heat - you can easily adjust the spice level to your liking.","author":"Liam Chen","date":"2023-05-05","category":"Main Course","tags":["thai","vegetarian","curry","spicy"],"rating":4.6}
+{"index":{"_id":"3"}}
+{"title":"Classic Beef Stroganoff: A Creamy Comfort Food","description":"Indulge in this rich and creamy beef stroganoff. Tender strips of beef in a savory mushroom sauce, served over a bed of egg noodles. It's the ultimate comfort food for chilly evenings.","author":"Emma Watson","date":"2023-05-10","category":"Main Course","tags":["beef","pasta","comfort food"],"rating":4.7}
+{"index":{"_id":"4"}}
+{"title":"Vegan Chocolate Avocado Mousse","description":"Discover the magic of avocado in this rich, vegan chocolate mousse. Creamy, indulgent, and secretly healthy, it's the perfect guilt-free dessert for chocolate lovers.","author":"Alex Green","date":"2023-05-15","category":"Dessert","tags":["vegan","chocolate","avocado","healthy dessert"],"rating":4.5}
+{"index":{"_id":"5"}}
+{"title":"Crispy Oven-Fried Chicken","description":"Get that perfect crunch without the deep fryer! This oven-fried chicken recipe delivers crispy, juicy results every time. A healthier take on the classic comfort food.","author":"Maria Rodriguez","date":"2023-05-20","category":"Main Course","tags":["chicken","oven-fried","healthy"],"rating":4.9}
+----
+// TEST
+
+[discrete]
+[[full-text-filter-tutorial-match-query]]
+=== Step 3: Perform basic full-text searches
+
+Full-text search involves executing text-based queries across one or more document fields.
+These queries calculate a relevance score for each matching document, based on how closely the document's content aligns with the search terms.
+{es} offers various query types, each with its own method for matching text and <<relevance-scores,relevance scoring>>.
+
+[discrete]
+==== `match` query
+
+The <<query-dsl-match-query, `match`>> query is the standard query for full-text, or "lexical", search.
+The query text will be analyzed according to the analyzer configuration specified on each field (or at query time).
+
+First, search the `description` field for "fluffy pancakes":
+
+[source,console]
+----
+GET /cooking_blog/_search
+{
+  "query": {
+    "match": {
+      "description": {
+        "query": "fluffy pancakes", <1>
+        "analyzer": "standard" <2>
+      }
+    }
+  }
+}
+----
+// TEST[skip:Flakey test]
```

Comment:
Why is this test flakey?

Reply:
rerunning and will check

Reply:
<details>
  <summary>Expand to see build fail</summary>



```	FAILURE: Build failed with an exception.
2024-10-23 13:35:56 CEST	
2024-10-23 13:35:56 CEST	* What went wrong:
2024-10-23 13:35:56 CEST	Execution failed for task ':docs:yamlRestTest'.
2024-10-23 13:35:56 CEST	> There were failing tests. See the report at: file:///dev/shm/bk/bk-agent-prod-gcp-1729682825607451781/elastic/elasticsearch-pull-request/docs/build/reports/tests/yamlRestTest/index.html
2024-10-23 13:35:56 CEST	
2024-10-23 13:35:56 CEST	* Exception is:
2024-10-23 13:35:56 CEST	org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':docs:yamlRestTest'.
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:130)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.Try$Failure.ifSuccessfulOrElse(Try.java:293)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:128)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:116)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:74)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:209)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:166)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:52)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:42)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:331)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:318)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.lambda$execute$0(DefaultTaskExecutionGraph.java:314)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.CurrentBuildOperationRef.with(CurrentBuildOperationRef.java:85)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:314)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:303)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:459)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:376)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.concurrent.AbstractManagedExecutor$1.run(AbstractManagedExecutor.java:48)
2024-10-23 13:35:56 CEST	        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
2024-10-23 13:35:56 CEST	        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
2024-10-23 13:35:56 CEST	        at java.base/java.lang.Thread.run(Thread.java:1583)
2024-10-23 13:35:56 CEST	Caused by: org.gradle.api.internal.exceptions.MarkedVerificationException: There were failing tests. See the report at: file:///dev/shm/bk/bk-agent-prod-gcp-1729682825607451781/elastic/elasticsearch-pull-request/docs/build/reports/tests/yamlRestTest/index.html
2024-10-23 13:35:56 CEST	        at org.gradle.api.tasks.testing.AbstractTestTask.handleTestFailures(AbstractTestTask.java:706)
2024-10-23 13:35:56 CEST	        at org.gradle.api.tasks.testing.AbstractTestTask.handleCollectedResults(AbstractTestTask.java:540)
2024-10-23 13:35:56 CEST	        at org.gradle.api.tasks.testing.AbstractTestTask.executeTests(AbstractTestTask.java:535)
2024-10-23 13:35:56 CEST	        at org.gradle.api.tasks.testing.Test.executeTests(Test.java:731)
2024-10-23 13:35:56 CEST	        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
2024-10-23 13:35:56 CEST	        at java.base/java.lang.reflect.Method.invoke(Method.java:580)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:125)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:58)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:51)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:29)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.TaskExecution$3.run(TaskExecution.java:244)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:166)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:47)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.TaskExecution.executeAction(TaskExecution.java:229)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.TaskExecution.executeActions(TaskExecution.java:212)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.TaskExecution.executeWithPreviousOutputFiles(TaskExecution.java:195)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.TaskExecution.execute(TaskExecution.java:162)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ExecuteStep.executeInternal(ExecuteStep.java:105)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ExecuteStep.access$000(ExecuteStep.java:44)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ExecuteStep$1.call(ExecuteStep.java:59)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ExecuteStep$1.call(ExecuteStep.java:56)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:209)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:166)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ExecuteStep.execute(ExecuteStep.java:56)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ExecuteStep.execute(ExecuteStep.java:44)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.CancelExecutionStep.execute(CancelExecutionStep.java:42)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.TimeoutStep.executeWithoutTimeout(TimeoutStep.java:75)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.TimeoutStep.execute(TimeoutStep.java:55)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.PreCreateOutputParentsStep.execute(PreCreateOutputParentsStep.java:50)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.PreCreateOutputParentsStep.execute(PreCreateOutputParentsStep.java:28)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.RemovePreviousOutputsStep.execute(RemovePreviousOutputsStep.java:67)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.RemovePreviousOutputsStep.execute(RemovePreviousOutputsStep.java:37)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.BroadcastChangingOutputsStep.execute(BroadcastChangingOutputsStep.java:61)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.BroadcastChangingOutputsStep.execute(BroadcastChangingOutputsStep.java:26)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.CaptureOutputsAfterExecutionStep.execute(CaptureOutputsAfterExecutionStep.java:69)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.CaptureOutputsAfterExecutionStep.execute(CaptureOutputsAfterExecutionStep.java:46)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ResolveInputChangesStep.execute(ResolveInputChangesStep.java:40)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ResolveInputChangesStep.execute(ResolveInputChangesStep.java:29)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.BuildCacheStep.executeWithoutCache(BuildCacheStep.java:189)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.BuildCacheStep.executeAndStoreInCache(BuildCacheStep.java:145)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.BuildCacheStep.lambda$executeWithCache$4(BuildCacheStep.java:101)
2024-10-23 13:35:56 CEST	        at java.base/java.util.Optional.orElseGet(Optional.java:364)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.BuildCacheStep.lambda$executeWithCache$5(BuildCacheStep.java:101)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.Try$Success.map(Try.java:175)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.BuildCacheStep.executeWithCache(BuildCacheStep.java:85)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.BuildCacheStep.lambda$execute$0(BuildCacheStep.java:74)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.Either$Left.fold(Either.java:115)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.caching.CachingState.fold(CachingState.java:62)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.BuildCacheStep.execute(BuildCacheStep.java:73)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.BuildCacheStep.execute(BuildCacheStep.java:48)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.StoreExecutionStateStep.execute(StoreExecutionStateStep.java:46)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.StoreExecutionStateStep.execute(StoreExecutionStateStep.java:35)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.SkipUpToDateStep.executeBecause(SkipUpToDateStep.java:75)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.SkipUpToDateStep.lambda$execute$2(SkipUpToDateStep.java:53)
2024-10-23 13:35:56 CEST	        at java.base/java.util.Optional.orElseGet(Optional.java:364)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:53)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:35)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsFinishedStep.execute(MarkSnapshottingInputsFinishedStep.java:37)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsFinishedStep.execute(MarkSnapshottingInputsFinishedStep.java:27)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ResolveIncrementalCachingStateStep.executeDelegate(ResolveIncrementalCachingStateStep.java:49)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ResolveIncrementalCachingStateStep.executeDelegate(ResolveIncrementalCachingStateStep.java:27)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.AbstractResolveCachingStateStep.execute(AbstractResolveCachingStateStep.java:71)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.AbstractResolveCachingStateStep.execute(AbstractResolveCachingStateStep.java:39)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ResolveChangesStep.execute(ResolveChangesStep.java:65)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ResolveChangesStep.execute(ResolveChangesStep.java:36)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ValidateStep.execute(ValidateStep.java:107)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ValidateStep.execute(ValidateStep.java:56)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.AbstractCaptureStateBeforeExecutionStep.execute(AbstractCaptureStateBeforeExecutionStep.java:64)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.AbstractCaptureStateBeforeExecutionStep.execute(AbstractCaptureStateBeforeExecutionStep.java:43)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.AbstractSkipEmptyWorkStep.executeWithNonEmptySources(AbstractSkipEmptyWorkStep.java:125)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.AbstractSkipEmptyWorkStep.execute(AbstractSkipEmptyWorkStep.java:61)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.AbstractSkipEmptyWorkStep.execute(AbstractSkipEmptyWorkStep.java:36)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsStartedStep.execute(MarkSnapshottingInputsStartedStep.java:38)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.LoadPreviousExecutionStateStep.execute(LoadPreviousExecutionStateStep.java:36)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.LoadPreviousExecutionStateStep.execute(LoadPreviousExecutionStateStep.java:23)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.HandleStaleOutputsStep.execute(HandleStaleOutputsStep.java:75)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.HandleStaleOutputsStep.execute(HandleStaleOutputsStep.java:41)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.AssignMutableWorkspaceStep.lambda$execute$0(AssignMutableWorkspaceStep.java:35)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.TaskExecution$4.withWorkspace(TaskExecution.java:289)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.AssignMutableWorkspaceStep.execute(AssignMutableWorkspaceStep.java:31)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.AssignMutableWorkspaceStep.execute(AssignMutableWorkspaceStep.java:22)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ChoosePipelineStep.execute(ChoosePipelineStep.java:40)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ChoosePipelineStep.execute(ChoosePipelineStep.java:23)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ExecuteWorkBuildOperationFiringStep.lambda$execute$2(ExecuteWorkBuildOperationFiringStep.java:67)
2024-10-23 13:35:56 CEST	        at java.base/java.util.Optional.orElseGet(Optional.java:364)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ExecuteWorkBuildOperationFiringStep.execute(ExecuteWorkBuildOperationFiringStep.java:67)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.ExecuteWorkBuildOperationFiringStep.execute(ExecuteWorkBuildOperationFiringStep.java:39)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.IdentityCacheStep.execute(IdentityCacheStep.java:46)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.IdentityCacheStep.execute(IdentityCacheStep.java:34)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.IdentifyStep.execute(IdentifyStep.java:48)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.steps.IdentifyStep.execute(IdentifyStep.java:35)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.execution.impl.DefaultExecutionEngine$1.execute(DefaultExecutionEngine.java:61)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:127)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:116)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:74)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:209)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:166)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
2024-10-23 13:35:56 CEST	        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:52)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:42)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:331)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:318)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.lambda$execute$0(DefaultTaskExecutionGraph.java:314)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.operations.CurrentBuildOperationRef.with(CurrentBuildOperationRef.java:85)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:314)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:303)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:459)
2024-10-23 13:35:56 CEST	        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:376)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
2024-10-23 13:35:56 CEST	        at org.gradle.internal.concurrent.AbstractManagedExecutor$1.run(AbstractManagedExecutor.java:48)
2024-10-23 13:35:56 CEST	        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
2024-10-23 13:35:56 CEST	        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
2024-10-23 13:35:56 CEST	        at java.base/java.lang.Thread.run(Thread.java:1583)
2024-10-23 13:35:56 CEST```

</details>